test/grub_install_test.rb

```diff
@@ -0,0 +1,132 @@
+#! /usr/bin/env rspec --format doc
+
+require_relative "./test_helper"
+
+require "bootloader/grub_install"
+
+describe Bootloader::GrubInstall do
+  describe "#execute" do
+    def stub_arch(arch)
```

Comment:
Actually I have version with let and I do not like much output. I think nesting make sense where there is at least two stements in context. Something like
```
on s390
  it propose this
  it propose that
  it do that
```
for simple stement in this case it create something like
```
on i386
  use i386-efi
on x86_64
  use x86_64-efi
...
```
where I like more single like statement as it is more like mapping from arch to grub2 target.